web/src/common.ts

```diff
@@ -162,4 +162,11 @@ export function setup_password_visibility_toggle(
         e.stopPropagation();
         toggle_password_visibility(password_field_id, password_selector, tippy_tooltips);
     });
+    $(password_selector).on("keyup", (e) => {
```

Comment:
That’s how keyboards work, and users know that, so they aren’t going to be holding the key for that long. Use `keydown`.

Or better, use an actual `<button>` element, for which `Enter` automatically fires the `click` event.